schema/library.schema.json

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "properties": {
+        "library_item_id": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 67
+        },
+        "description_brief": {
+            "type": "string"
+        },
+        "description_detail": {
+            "type": "string"
+        },
+        "description_index": {
+            "type": "array",
+            "items": {
+                "type": "string"
+            }
+        },
+        "description_datapoints": {
+            "type": "object"
+        },
+        "description_verification_type": {
+            "enum": [
+                "rule-based",
+                "procedure-based"
```

Comment:
I'm fine with just these for now since that we don't have any `ml-based` one yet. But we might want to add it eventually.